gradle/validation/error-prone.gradle

```diff
@@ -74,17 +74,14 @@ allprojects { prj ->
             '-Xep:StaticAssignmentInConstructor:OFF', // we assign SolrTestCaseJ4.configString in many tests, difficult to untangle
             '-Xep:ComparableType:OFF', // SolrTestCaseJ4.Doc and Fld are messy
 
-            '-Xep:AlmostJavadoc:OFF',
```

Comment:
I like that in Rubocop, each rule is added to the list and is either set to on or off, so you can see everything you are enforcing, versus having things that aren't listed as being ON.   This is totally a stylistic thing...

Reply:
https://errorprone.info/bugpatterns

there is a very big list and grows/changes each time errorprone is updated.